lib/generation/generators/value_objects/template_strategy/stateless_template_strategy.dart

```diff
@@ -18,6 +19,7 @@ class StatelessTemplateStrategy extends TemplateStrategy {
     var overrides = '';
     var overrideVars = '';
 
+    print('${PBSymbolStorage().sharedMasterNodes.length}');
```

Comment:
Hello @GVTopCoder, I believe this print statement may have been left there on accident!